pint_xarray/tests/test_conversion.py

```diff
@@ -176,6 +180,29 @@ def test_attach_units(self, obj, units):
 
         assert conversion.extract_units(actual) == units
 
+    @pytest.mark.parametrize(
+        ["obj", "units"],
+        (
+            pytest.param(
+                DataArray(dims="x", coords={"x": [], "u": ("x", [])}),
+                {None: "hPa", "x": "m"},
+                id="DataArray",
+            ),
+            pytest.param(
+                Dataset(
+                    data_vars={"a": ("x", []), "b": ("x", [])},
+                    coords={"x": [], "u": ("x", [])},
+                ),
+                {"a": "degK", "b": "hPa", "u": "m"},
```

Comment:
This really shouldn't bother me, but it does :wink:. Even though Pint allows it, it just feels wrong to say "degrees Kelvin"...it's just "Kelvin"

```suggestion
                {"a": "K", "b": "hPa", "u": "m"},
```